docs/welcome.md

```diff
@@ -0,0 +1,49 @@
+----
+-title: Welcome
+-description: Getting started
+-keywords: .NET
+-author: runfaster2000
+-manager: wpickett
+-ms.date: 06/23/2016
+-ms.topic: article
+-ms.prod: .net
+-ms.technology: .net-technologies
+-ms.devlang: dotnet
+-ms.assetid: cb788dcf-2120-467f-9c34-c02a90e1f68f
+----
```

Comment:
Preview is not rendering the metadata table correctly. I believe it's only 3 dashes here.